tools/wptrunner/wptrunner/executors/base.py

```diff
@@ -357,12 +358,13 @@ class RefTestExecutor(TestExecutor):
     is_print = False
 
     def __init__(self, logger, browser, server_config, timeout_multiplier=1, screenshot_cache=None,
-                 debug_info=None, **kwargs):
+                 debug_info=None, reftest_screenshot=None, **kwargs):
```

Comment:
Let's default to "unexpected" since in practice this can't actually have a `None` value (we always set it in wptcommandline.py, with the no-argument default being set depending on which other parameters were supplied).

Reply:
done